homeworks/nikita.karchinskiy_Nikkarchinskiy/4-moduls/exports.js

```diff
@@ -0,0 +1,52 @@
+const monthIs = document.querySelector('[data-mounth-long-answer]');
+const monthMondays = document.querySelector('[data-mondays-answer]');
+const shortWeeks = document.querySelector('[data-shortest-week-answer]');
+
+export function daysInMonthFnc(isYear, isMonth) {
+    const daysInMonth = new Date(isYear, isMonth, 0).getDate();
+    if (daysInMonth < 31) {
+        monthIs.insertAdjacentHTML('beforeend', 'long');
```

Comment:
Why do you use `insertAdjacentHTML`? This method adds content to the current. So each time you select a month and press a button, your `monthIs` gets bigger.

I consider using `innerText` for this case.

Reply:
Also, this function should write nothing directly to DOM. The role of the function: receive the date and return is this a long month.

You need to move all DOM manipulations into the `index.js` file.